gtk/gtk.go

```diff
@@ -8516,6 +8516,62 @@ func (v *TreeModelFilter) ConvertIterToChildIter(filterIter *TreeIter) *TreeIter
 	return t
 }
 
+// ConvertPathToChildPath is a wrapper around gtk_tree_model_filter_convert_path_to_child_path().
+func (v *TreeModelFilter) ConvertPathToChildPath(filterPath *TreePath) *TreePath {
+	path := C.gtk_tree_model_filter_convert_path_to_child_path(v.native(), filterPath.native())
+	if path == nil {
+		return nil
+	}
+
+	p := &TreePath{path}
+	runtime.SetFinalizer(p, (*TreePath).free)
```

Comment:
Is manually freeing memory actually necessary?

Reply:
The GTK documentation for [gtk_tree_model_filter_convert_path_to_child_path](https://developer.gnome.org/gtk3/stable/GtkTreeModelFilter.html#gtk-tree-model-filter-convert-path-to-child-path) says that it returns a newly allocated GtkTreePath with "transfer full" meaning free the data after done. If this isn't the right thing to do in the wrapper, I'll take that line out.

Reply:
To be honest, I don't know much about this topic, which is why I was wondering. Maybe @andre-hub could enlighten me about what the `garbage collection has been taken care of` phrase in the README refers to exactly. Maybe we can clarify that somewhat, since I feel like I am not the only one who was ever confused about this topic.